.travis.yml

```diff
@@ -0,0 +1,9 @@
+sudo: required
+language: python
+env: PGVERSION=9.1
+install:
+    - pip install -r dev-requirements.txt
```

Comment:
Why the choice to call it `dev-requirements.txt`?

Reply:
@DavidLemayian this is because it contains the dependencies a developer will need when testing. Then the requirements.txt contains dependencies unique to this particular extension

Reply:
@WNjihia I was wondering about the choice of file name. i.e why `dev-requirements.txt` instead of `requirements-test.txt`? Have you seen the naming convention being used in other projects for test requirements file?

Reply:
@WNjihia good idea. It looks like ckanext-harvest actually uses bash scripts to do the setup for CKAN et al. the `dev-requirements.txt` are for any development (non-production) requirements:

1. https://github.com/ckan/ckanext-harvest/tree/master/bin
2. https://github.com/ckan/ckanext-harvest/blob/master/.travis.yml

Reply:
@DavidLemayian I have looked at the bash scripts and they do the same thing as the `.travis.yml`. Do you suggest we use the scripts instead?